CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Your contribution here.
 * [#1832](https://github.com/ruby-grape/grape/pull/1832): Support `body_name` in `desc` block - [@fotos](https://github.com/fotos).
 * [#1831](https://github.com/ruby-grape/grape/pull/1831): Support `security` in `desc` block - [@fotos](https://github.com/fotos).
+* [#1831](https://github.com/ruby-grape/grape/pull/1833): Allows to set the `ParamBuilder` globally - [@myxoh](https://github.com/myxoh).
```

Comment:
The PR is 1833. Maybe order contributions in desc order?

Reply:
You are right, my bad